### ISSUE TITLE
Invalid hash syntax

### ISSUE THREAD

**Issue by matkowal** [westonganger/spreadsheet_architect#7]
After including the gem into Gemfile I started receiving syntax errors:
```
/Users/john/.rvm/gems/ruby-2.1.5@railsapp/gems/bundler-1.11.2/lib/bundler/runtime.rb:77:in `require': /Users/john/.rvm/gems/ruby-2.1.5@railsapp/gems/spreadsheet_architect-1.4.5/lib/spreadsheet_architect.rb:215: syntax error, unexpected ':', expecting keyword_end (SyntaxError)
            property :text, 'font-weight': :bold
                                          ^
/Users/john/.rvm/gems/ruby-2.1.5@railsapp/gems/spreadsheet_architect-1.4.5/lib/spreadsheet_architect.rb:218: syntax error, unexpected ':', expecting keyword_end
            property :text, 'align': options[:header_style][:align]
                                    ^
/Users/john/.rvm/gems/ruby-2.1.5@railsapp/gems/spreadsheet_architect-1.4.5/lib/spreadsheet_architect.rb:221: syntax error, unexpected ':', expecting keyword_end
            property :text, 'font-size': options[:header_style][:size]
                                        ^
/Users/john/.rvm/gems/ruby-2.1.5@railsapp/gems/spreadsheet_architect-1.4.5/lib/spreadsheet_architect.rb:224: syntax error, unexpected ':', expecting keyword_end
...       property :text, 'color': "##{options[:header_style][:...
...                               ^
/Users/john/.rvm/gems/ruby-2.1.5@railsapp/gems/spreadsheet_architect-1.4.5/lib/spreadsheet_architect.rb:231: syntax error, unexpected ':', expecting keyword_end
            property :text, 'font-weight': :bold
                                          ^
/Users/john/.rvm/gems/ruby-2.1.5@railsapp/gems/spreadsheet_architect-1.4.5/lib/spreadsheet_architect.rb:234: syntax error, unexpected ':', expecting keyword_end
            property :text, 'align': options[:row_style][:align]
                                    ^
/Users/john/.rvm/gems/ruby-2.1.5@railsapp/gems/spreadsheet_architect-1.4.5/lib/spreadsheet_architect.rb:237: syntax error, unexpected ':', expecting keyword_end
            property :text, 'font-size': options[:row_style][:size]
                                        ^
/Users/john/.rvm/gems/ruby-2.1.5@railsapp/gems/spreadsheet_architect-1.4.5/lib/spreadsheet_architect.rb:240: syntax error, unexpected ':', expecting keyword_end
            property :text, 'color': "##{options[:row_style][:color]}"




```

**Comment by westonganger** [comment]
This is the ruby 2.1+ hash syntax, I will fix it so it still works with 1.9.3+

**Comment by westonganger** [comment]
I just release v1.4.6 which fixes this issue.
